src/device-registry/models/Site.js

```diff
@@ -162,6 +162,28 @@ const siteSchema = new Schema(
       type: String,
       trim: true,
     },
+    nearest_tahmo_station: {
+      id: {
```

Comment:
Might we need some defaults, or it should not be a worry? @NoahNsimbe

Reply:
@Baalmart, I have added them